tests/unit/test_outlook.py

```diff
@@ -0,0 +1,94 @@
+import os
+import pytest
+from unittest import mock
+from datetime import datetime
+
+import pandas as pd
+from O365.account import Account
+
+from viadot.sources import Outlook
+from viadot.exceptions import CredentialError
+
+
+@pytest.fixture
+def var_dictionary():
+    variables = {
+        "mailbox_name": "vevoszolgalat@velux.com",
```

Comment:
I'm not sure if the real mailbox should be here. Use fake data if possible.

Reply:
changed!